### PR TITLE
Order Items

### PR DESCRIPTION
**Pull request by Isterius**

### CODE REVIEW

File: server/graphql/queries/order.ts [hunk 1]
```diff
@@ -21,7 +21,6 @@ export default {
     description: 'Get order by ID',
     args: { id: { type: GraphQLID } },
     resolve: (root, {id}) => {
-      return OrderModel.findById(new mongoose.Types.ObjectId(id));
-    }
+      return OrderModel.findById(new mongoose.Types.ObjectId(id)).populate('user').populate('orderItems').exec();    }
```
Comment (pawelkondraciuk) [inline server/graphql/queries/order.ts]:
```
resolve: (root, {id}) => {
  const orderId = new mongoose.Types.ObjectId(id);
  return OrderModel
    .findById(orderId)
    .populate('user')
    .populate('orderItems').exec();
}
```

Looks way better

File: src/app/shared/services/orders.service.ts [hunk 2]
```diff
@@ -57,7 +60,7 @@ export class OrdersService {
     });
   }
 
-  public getById(orderId: string): any {
+  public getById(orderId: string): Observable<any> {
```
Comment (pawelkondraciuk) [inline src/app/shared/services/orders.service.ts]: You're lacking query type

File: src/app/shared/services/orders.service.ts [hunk 3]
```diff
@@ -68,6 +71,17 @@ export class OrdersService {
     .switchMap((res) => Observable.of(res.data.order));
   }
 
+  public getOrderDetails(orderId: string): Observable<any> {
```
Comment (pawelkondraciuk) [inline src/app/shared/services/orders.service.ts]: Same here

**Comment by Isterius** [comment]
@pawelkondraciuk pls cr my friend

**Review by pawelkondraciuk — APPROVED** [review]

**Pull request merged**